frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -350,7 +349,7 @@
       >
```

Comment:
@rmunn, I was wondering if I am supposed to add a listener here for it to be able to render the change without having to refresh the page.